lib/phoenix/controller.ex

```diff
@@ -578,7 +587,16 @@ defmodule Phoenix.Controller do
     put_private_view(conn, :phoenix_view, :new, formats)
   end
 
-  def put_new_view(%Plug.Conn{}, _module), do: raise(AlreadySentError)
+  def put_new_view(%Plug.Conn{} = conn, module) do
+    raise(AlreadySentError, """
+      The response was already sent.
+
+          Status code: `#{conn.status}`
+          Request path: `#{conn.request_path}`
+          Method: `#{conn.method}`
+          View module: `#{inspect(module)}`
```

Comment:
Error messages start in lowercase and I think the backticks would make things harder to read, so I suggest removing them:

```suggestion
      the response was already sent.

          Status code: #{conn.status}
          Request path: #{conn.request_path}
          Method: #{conn.method}
          View module: #{inspect(module)}
```